index.html

```diff
@@ -35,12 +35,144 @@ <h3>I am a Software Developer</h3>
             <ul>
                 <li><a href="#"><i class="bi bi-github"></i></a></li>
                 <li><a href="#"><i class="bi bi-linkedin"></i></a></li>
-                <li><a href="#"><img src="NormalButton/Tertiary/Icons/Vector.svg" alt="peace button social media"></a>
+                <li><a href="#"><img src="vectorsImg/Tertiary/Icons/Vector.svg" alt="peace button social media"></a>
                 </li>
                 <li><a href="#"><i class="bi bi-twitter"></i></a></li>
-                <li><a href="#"><img src="NormalButton/Tertiary/Icons/Mvector.svg" alt="m button social media"></a></li>
+                <li><a href="#"><img src="vectorsImg/Tertiary/Icons/Mvector.svg" alt="m button social media"></a></li>
             </ul>
         </section>
+
+        <section class="work-section">
+            <h2>My Recent Works </h2>
+
+            <article class="card">
+                <img src="images/ImgProj.png" alt="Project image">
+                <h3>Multi-Post Stories</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.</p>
+                <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#" class="first-btn">See Project</a>
+            </article>
+
+            <article class="card card-bg">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                <a href="#" class="fullWidth-btn">See Project</a>
```

Comment:
Please fix the aligning here 

| Your design | Figma |
| ------------- | ------- |
| ![image](https://user-images.githubusercontent.com/74744787/164718810-33f96c04-4e6f-489f-b329-53d6af918a63.png) | ![image](https://user-images.githubusercontent.com/74744787/164718870-0a5d62a1-b4a9-4d67-9b10-221782667382.png) |

to fix this I suggest: 
- add align item center to your section aligned with grid 
![image](https://user-images.githubusercontent.com/74744787/164719298-289215a4-88cf-46bc-bd14-45c43158706f.png)

- change the width of this image 
![image](https://user-images.githubusercontent.com/74744787/164719398-c80cb429-d2d0-4442-b645-7b7c39525840.png)

- change the font size of this title 
![image](https://user-images.githubusercontent.com/74744787/164719559-bbf3caf9-2472-4f64-ba88-4b2b279489f3.png)